features/FEATURE_BLE/ble/SafeEnum.h

```diff
@@ -126,71 +126,71 @@ struct SafeEnum {
      * @return true if the inner value of lhs and rhs are equal and false
      * otherwise.
      */
-	friend bool operator==(SafeEnum lhs, SafeEnum rhs) {
+	friend bool operator==(Target lhs, Target rhs) {
 		return lhs._value == rhs._value;
 	}
 
     /**
-     * Not equal to operator for SafeEnum instances.
+     * Not equal to operator for Target instances.
```

Comment:
Missed this search-and-replace on 'operator=='?

Reply:
Fixed